centreon/tests/e2e/features/Dashboards/18-web-page-widget-configuration.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Configuring web page widget
```

Comment:
I see that sometimes the word “**web**” and “**Web**” are capitalized, is that normal? If not, please put everything in the same typography.